### PR TITLE
add unit test for metaserver util.go

### PR DESCRIPTION
**Pull request by TianTianBigWang**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/metaserver/util/util_test.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"reflect"
+	"testing"
```
Comment (Iceber) [inline pkg/metaserver/util/util_test.go:14]: format the import package.

**Comment by fisherxu** [comment]
Please get the ci passed, thanks :)

**Comment by Iceber** [comment]
@TianTianBigWang please combine into one commit.

**Comment by TianTianBigWang** [comment]
> @TianTianBigWang please combine into one commit.

Done.

**Pull request merged**